src/analyze/analyze.c

```diff
@@ -1551,6 +1553,29 @@ static int dump_syscall_filters(int argc, char *argv[], void *userdata) {
 }
 #endif
 
+static int dump_timespan(int argc, char *argv[], void *userdata) {
+        int r;
+        char **input_timespan;
+        usec_t usec_magnitude = 1, output_usecs;
+        char ft_buf[FORMAT_TIMESPAN_MAX];
+
+        STRV_FOREACH(input_timespan, strv_skip(argv, 1)) {
+            r = parse_time(*input_timespan, &output_usecs, usec_magnitude);
+            if (r < 0) {
+                    return log_error_errno(r, "Failed to parse time span '%s': %m", *input_timespan);
+            }
+
+            printf("Original: %s\n", *input_timespan);
+            printf("      μs: %" PRIu64 "\n", output_usecs);
```

Comment:
I line the use of unicode, but I figure this needs to be added to the special_glyph() logic, see locale-util.[ch] for details, so that it has an ASCII fallback (µ → u)...